app/services/stories_api/v3/endpoints/featured.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+# The majority of the Supplejack API code is Crown copyright (C) 2014, New Zealand Government,
+# and is licensed under the GNU General Public License, version 3.
+# One component is a third party component. See https://github.com/DigitalNZ/supplejack_api for details.
+#
+# Supplejack was created by DigitalNZ at the National Library of NZ and
+# the Department of Internal Affairs. http://digitalnz.org/supplejack
+
+module StoriesApi
+  module V3
+    module Endpoints
+      class Featured
+        include Helpers
+        attr_reader :params, :errors
```

Comment:
Why do you have these attribute readers? I can't see where the values get created.